data/org.freedesktop.portal.SpeechSynthesis.xml

```diff
@@ -0,0 +1,169 @@
+
+        * ``session_handle_token`` (``s``)
+
+              A string that will be used as the last element of the session handle. Must be a valid
+              object path element. See the :ref:`org.freedesktop.portal.Session` documentation for
+              more information about the session handle.
+
+    -->
+    <method name="CreateSession">
+      <annotation name="org.qtproject.QtDBus.QtTypeName.In0" value="QVariantMap"/>
+      <arg type="a{sv}" name="options" direction="in"/>
+      <arg type="o" name="handle" direction="out"/>
+    </method>
+
+    <!--
+        GetProviders:
+        @session_handle: Object path for the :ref:`org.freedesktop.portal.Session` object
+        @parent_window: Identifier for the application window, see :doc:`window-identifiers`
+        @handle: Object path for the :ref:`org.freedesktop.portal.Request` object representing this call
+
+        Get available synthesis voices
+
+        Supported keys in the @options vardict include:
+
+        * ``handle_token`` (``s``)
+
+          A string that will be used as the last element of the @handle. Must be a valid
+          object path element. See the :ref:`org.freedesktop.portal.Request` documentation for
+          more information about the @handle.
+
+        The following results get returned via the :ref:`org.freedesktop.portal.Request::Response` signal:
+
+        * ``providers`` (``a(ss)``)
+
+           An array of providers. Each provider in the array is structure with the following members:
+             * A well known name
+             * A human readable name
+    -->
+    <method name="GetProviders">
+      <arg type="o" name="session_handle" direction="in"/>
+      <arg type="s" name="parent_window" direction="in"/>
+      <arg type="a{sv}" name="options" direction="in"/>
+      <arg type="o" name="handle" direction="out"/>
+    </method>
+
+    <!--
+        GetVoices:
+        @session_handle: Object path for the :ref:`org.freedesktop.portal.Session` object
+        @parent_window: Identifier for the application window, see :doc:`window-identifiers`
+        @handle: Object path for the :ref:`org.freedesktop.portal.Request` object representing this call
+
+        Get available synthesis voices
+
+        Supported keys in the @options vardict include:
+
+        * ``handle_token`` (``s``)
+
+          A string that will be used as the last element of the @handle. Must be a valid
+          object path element. See the :ref:`org.freedesktop.portal.Request` documentation for
+          more information about the @handle.
+
+        The following results get returned via the :ref:`org.freedesktop.portal.Request::Response` signal:
+
+        * ``voices`` (``a(ssstas)``)
+
+          An array of voices. Each voice in the array is structure with the following members:
+            * A human readable name
+            * A unique identifier
+            * Synthesis output format
+            * A voice features bit field
```

Comment:
any chance the bit field could be expanded with it possible values?

Reply:
Yeah, I can add that. Will do that on monday.